modules/xmpp/SDP.js

```diff
@@ -200,9 +199,9 @@ SDP.prototype.toJingle = function (elem, thecreator) {
 
                 elem.up();
             }
-            if (SDPUtil.find_line(this.media[i], 'a=crypto:', this.session)) {
+            var crypto = SDPUtil.find_lines(this.media[i], 'a=crypto:', this.session);
+            if (crypto.length) {
```

Comment:
I'm afraid I don't get what you mean here.

I'd rather have this merged as soon as possible because it fixes a particular problem: searching for one and the same substring in a string.